README.md

```diff
@@ -1,42 +1,46 @@
-# SpiceDB
-
+SpiceDB
```

Comment:
was this intentional?

Reply:
Yeah, this makes the badges on the same line, but yeah the linter hates mixing these styles of headings in markdown